linode.gemspec

```diff
@@ -2,8 +2,7 @@
 
 Gem::Specification.new do |s|
   s.name = "linode"
-  s.version = "0.9.0"
-  s.date = "2016-01-02"
+  s.version = File.open("VERSION", "r").read.strip
```

Comment:
No reason to have to update this and the VERSION file every time you make a version bump, just read the file.